Makefile

```diff
@@ -7,10 +7,17 @@ BINDIR ?= $(PREFIX)/bin
 VERSION="0.3.1"
 
 CFLAGS ?= -Wall -Wextra -Wno-unused-parameter -Wno-parentheses
+CFLAGS += `pkg-config --cflags xkbcommon` \
+	`pkg-config --cflags wayland-client` \
+	`pkg-config --cflags wayland-cursor`
```

Comment:
This needs to override, otherwise it won't build if somebody specifies "CFLAGS=-O2" or whatever on the command line.

Reply:
Actually, I'm not sure about this one. I'll have to check what the convention is.